v2/pkg/runner/runner.go

```diff
@@ -523,12 +608,13 @@ func (r *Runner) SetInterface(interfaceName string) error {
 	return nil
 }
 
-func (r *Runner) handleOutput() {
+func (r *Runner) handleOutput(scanResults *result.Result) {
```

Comment:
maybe we should split this function into multiple individual function as it's a bit hard to read